packages/php/email-editor/src/Engine/class-settings-controller.php

```diff
@@ -215,4 +189,36 @@ public function translate_slug_to_font_size( string $font_size ): string {
 	public function translate_slug_to_color( string $color_slug ): string {
 		return $this->theme_controller->translate_slug_to_color( $color_slug );
 	}
+
+	/**
+	 * Method to initialize iframe assets.
+	 *
+	 * @return void
+	 */
+	private function init_iframe_assets(): void {
+		if ( ! empty( $this->iframe_assets ) ) {
+			return;
+		}
+
+		/*
+		 * We need to initialize these assets early because they are read from global variables $wp_styles and $wp_scripts
+		 * and in later WordPress page load pages they contain stuff we don't want (e.g. html for admin login popup)
+		 * in the post editor this is called directly in post.php.
+		 */
+		$this->iframe_assets = _wp_get_iframed_editor_assets();
```

Comment:
I think the comment is now a bit outdated. We no longer make sure it runs early. I think we can skip it, because it can be confusing as is.